hnn_core/network.py

```diff
@@ -547,12 +534,8 @@ def add_bursty_drive(self, name, *, tstart=0, tstart_std=0, tstop=None,
         seedcore : int
             Optional initial seed for random number generator (default: 2).
         """
-        sim_end_time = self.cell_response.times[-1]
-        if tstop is None:
-            tstop = sim_end_time
```

Comment:
where do we now convert `tstop=None` to an actual value, given that `tstop` defaults to `None`?

Reply:
this happens inside `simulate_dipole`

Reply:
Okay got it!